### DISCUSSION TITLE
Unable to find symbols for 10.0.22000.282....

### DISCUSSION THREAD

**Discussion by jrg63** [valinet/ExplorerPatcher#123] [General]
I'm on the release preview ring for Windows 11 Pro and a new update was applied this morning. I installed the DLL and followed the instructions but its not working. I get a pop up that says

![image](https://user-images.githubusercontent.com/25476144/137567740-517ed984-69cb-42fe-b89c-608247f3b50c.png)

Cannot find symbols for 10.0.22000.282

Any suggestions on how to resolve this?

**Comment by valinet** [comment]
Well, at first run, it has to download some data from Microsoft. Unless it does that, it won’t function. Make sure nothing blocks or interferes with that. You should see network activity after that message (in Task Manager for example) and then after some time when the download finishes, Explorer would restart and that’d be it, you’d see the old taskbar and from there on it starts working.

**Comment by eGOoOGle** [comment]
Hello ! I have the same problem as jrg63. Nothing is requested about the network activity. 
![image](https://user-images.githubusercontent.com/92626494/137581782-3f544950-8938-409c-8aa9-eb33c765c65a.png)

I have this version because i have registrated to the Windows insiders program.

**Comment by Tango4free** [comment]
Hello here the same
After I uninstalled the update, everything works again.
StartAllBack (Trail) runs without problems on my 2 PC, but you can't place the taskbar on the sides, which is why my favorite is the ExplorerPatcher
I think Valinet will manage it.
Thank you Valinet for your great work.

**Comment by BraINstinct0** [comment]
Can confirm this behavior as of now.

**Comment by valinet** [comment]
The problem is that Microsoft hasn't yet made the symbols available for 282 (http://msdl.microsoft.com/download/symbols/twinui.pcshell.pdb/73402D8892AF2DA2639F29A5D2F483911/twinui.pcshell.pdb should serve you a file, not a blank page). Once they do, everything will work again, probably. Explained in #129 as well.

**Comment by llandyw** [comment]
What I'd like to know is which items need symbol files.  This might make it easier to find out what the current version of the .pdb files are.

I just downloaded the symbols for Windows explorer, and there was no version mismatch (EP gets errors, but symchk doesn't)

Andy
